### PR TITLE
fix(instance): allow port-range in security groups

### PR DESCRIPTION
**Pull request by jerome-quere**

### CODE REVIEW

**Comment by Sh4d1** [comment]
@jerome-quere #354

**Comment by jerome-quere** [comment]
I add test + split the ip flatten bug in a speprate PR

File: scaleway/resource_instance_security_group.go [hunk 1]
```diff
@@ -368,7 +368,7 @@ func securityGroupRuleSchema() *schema.Resource {
 			},
 			"port_range": {
 				Type:        schema.TypeString,
-				Computed:    true,
+				Optional:    true,
```
Comment (kindermoumoute) [inline scaleway/resource_instance_security_group.go]: should this be conflit with port?

Comment (jerome-quere) [reply scaleway/resource_instance_security_group.go]: ConflictsWith do no work on neasterd schema

Comment (QuentinBrosse) [inline scaleway/resource_instance_security_group.go]: Why not `Optional` and `Computed`?

Comment (jerome-quere) [reply scaleway/resource_instance_security_group.go]: It is never computed only the one you use will  be store in state either `port` or `port_range` but never both

**Pull request merged**